### PR TITLE
Issues/107

### LINKED ISSUE #88: Picking file
Picking file and discarding causes app to crash

### LINKED ISSUE #107: OpenGL General issue
- [x] Cubes must be by default of size 5
- [x] Cubes must have smooth edges (add antialiasing)
- [x] Zero object must not be drawn at all (check if setting zero to nan is a solution)

### PR DESCRIPTION
**Pull request by LemurPwned**
Fixes #107 
Resolves #88 
Closes #91

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -13,6 +12,7 @@
 
 from Windows.ChooseWidget import ChooseWidget
 from Windows.PlayerWindow import PlayerWindow
+from Windows.Select import Select
```
Comment (pawelkulig1) [inline main.py]: Add this file please.

Comment (LemurPwned) [reply main.py]: Done

**Comment by LemurPwned** [comment]
I suggest merging in order of appearance so #97 should go first if that is ok

**Comment by LemurPwned** [comment]
Ok added better positioning but this is solely MacOs issue (verified both on Windows and  
Since such features are system agnostic (and so is pygame) I will simply disable text rendering for
Mac.

**Comment by pawelkulig1** [comment]
Nothing fixed for now. Nothing changed icon still the same.

**Comment by LemurPwned** [comment]
Ok, then text rendering will be disabled for Mac

**Comment by LemurPwned** [comment]
Just check repicking data now

**Comment by pawelkulig1** [comment]
Cannot insert text now but button for inserting is still active -should be disabled, but it's not so  important, resolve conflict and merge.

**Comment by LemurPwned** [comment]
Updating to fix the data repicking both for Linux and Mac and Windows.
Also, invalid namespace from merge updated

**Pull request merged**